### PR TITLE
web: mute the alert

### PR DESCRIPTION
**Pull request by 3pointer**
### What problem does this PR solve? 
github reports security issue, this PR try to mute it.

### What is changed and how it works?
update pakcage-lock.json in web

### Check List 

Tests 

 - No code

### Release note

 - No release note

### CODE REVIEW

File: web/package-lock.json [hunk 1]
```diff
@@ -1,2727 +1,8 @@
 {
   "name": "tidb-lightning-web",
   "version": "4.0.6",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (kennytm) [inline web/package-lock.json:4]: 🤔

File: web/package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "bignumber.js": "^9.0.0",
     "filesize": "^6.1.0",
     "json-bigint": "^1.0.0",
+    "lodash": "^4.17.21",
```
Comment (kennytm) [inline web/package.json:19]: ?

`npm update` should update the indirect deps automatically.

**Review by kennytm — APPROVED** [review]

**Review by YuJuncen — APPROVED** [review]

**Pull request merged**